app/js/components/common/Header.jsx

```diff
@@ -196,7 +196,7 @@ export default class Header extends Component {
 
   render() {
     return (
-      <div>
+      <div style={{...this.props.style, zIndex: 1}}>
```

Comment:
which styles are you passing via  props?

Reply:
This are the styles passed from the high node component `Sticky` which are used to make the header component sticky to the top of the page.

Reply:
Okay looks good